### DISCUSSION TITLE
Regarding social networks...

### DISCUSSION THREAD

**Discussion by jumaxotl** [waschinski/photo-stream#80] [Ideas]
Hello,

First and foremost, let me thank you for this excellent piece of minimalist perfection !

I would like to know though if it is possible/easy to make a change to the default social networks : instead of Twitter and Instagram, I'd like to set them to Mastodon (currently set as the custom link) and Pixelfed... Or maybe can I just add as many custom links as I want ?!

Cheers,

Julieñ

**Comment by waschinski** [comment]
Hi, adding support for Mastodon and Pixelfed links should be an easy task. What would be a good way to configure both links though? Looks like simply providing the whole URL seems to be a good idea, since usernames are unique on each server/instance only?

**Reply by jumaxotl** [reply]
Yes, this sounds right.

**Comment by waschinski** [comment]
This is currently in the dev branch and will be hitting the master soon, so it will be included in the next version. See #82.
